cob_twist_controller/include/cob_twist_controller/augmented_solver.h

```diff
@@ -56,26 +39,51 @@ class augmented_solver
      * default: 150
      *
      */
-    augmented_solver(const KDL::Chain& chain, double eps=0.001, int maxiter=5);
-    ~augmented_solver();
+    AugmentedSolver(const KDL::Chain& chain, double eps=0.001, int maxiter=5) :
+        chain_(chain),
+        jac_(chain_.getNrOfJoints()),
+        jnt2jac_(chain_),
+        maxiter_(maxiter),
+        x_(0.0),
+        y_(0.0),
+        r_(0.0),
+        z_(0.0)
```

Comment:
I can't find any usage of ```r_``` in the code -->remove

I think we can also get rid of x_, y_, z_ by setting 
```Eigen::Vector3d r_base_link(base_position.p.x(), base_position.p.y(), base_position.p.z());``` directly